src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -3130,6 +3164,25 @@ def aks_agentpool_add(cmd, client, resource_group_name, cluster_name, nodepool_n
     upgradeSettings = AgentPoolUpgradeSettings()
     taints_array = []
 
+    creationData = None
```

Comment:
We'd better load model with specific api version.
```suggestion
    # load model CreationData
    from azure.cli.command_modules.acs.decorator import AKSModels
    CreationData= AKSModels(cmd, ResourceType.MGMT_CONTAINERSERVICE).CreationData

    creationData = None
```